uat/pom.xml

```diff
@@ -23,22 +21,64 @@
             <url>https://d2jrmugq4soldf.cloudfront.net/snapshots</url>
         </repository>
     </repositories>
-
     <dependencies>
         <dependency>
             <groupId>com.aws.greengrass</groupId>
             <artifactId>aws-greengrass-testing-standalone</artifactId>
             <version>1.0-SNAPSHOT</version>
             <scope>compile</scope>
         </dependency>
-
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>cloudwatch</artifactId>
+            <version>2.18.13</version>
```

Comment:
OTF is using [sdk version](https://github.com/aws-greengrass/aws-greengrass-testing/blob/main/pom.xml#L40): 2.17.254
Just pointing out version differences.